src/main/java/com/microsoft/azure/proton/transport/ws/impl/WebSocketImpl.java

```diff
@@ -76,6 +76,24 @@ public WebSocketImpl() {
         isWebSocketEnabled = false;
     }
 
+    /**
+     * Create WebSocket transport layer - which, after configuring using
+     * the {@link #configure(String, String, String, int, String, Map, WebSocketHandler)} API
+     * is ready for layering in qpid-proton-j transport layers, using
+     * {@link org.apache.qpid.proton.engine.impl.TransportInternal#addTransportLayer(TransportLayer)} API.
+     * @param customMaxFrameSize the maximum frame size that this layer will buffer for
+     */
+    public WebSocketImpl(int customMaxFrameSize) {
+        inputBuffer = newWriteableBuffer(customMaxFrameSize);
```

Comment:
Could you make the default constructor call this one with that static final "maxFrameSize"? It would reduce some lines of duplication.

Reply:
sure thing!